tsl/test/expected/dist_util-15.out

```diff
@@ -972,17 +972,32 @@ SELECT 1 FROM pg_table_size('nondisttable');
         1
 (1 row)
 
--- hypertable_size requires SELECT privilege on table
-\set ON_ERROR_STOP 0
 SELECT * FROM hypertable_size('disttable');
-ERROR:  permission denied for table disttable
+ hypertable_size 
+-----------------
+          147456
+(1 row)
+
 SELECT * FROM hypertable_size('nondisttable');
-ERROR:  permission denied for table nondisttable
+ hypertable_size 
+-----------------
+          131072
+(1 row)
+
 SELECT * FROM hypertable_detailed_size('disttable') ORDER BY node_name;
-ERROR:  permission denied for table disttable
+ table_bytes | index_bytes | toast_bytes | total_bytes |  node_name  
+-------------+-------------+-------------+-------------+-------------
+       24576 |       49152 |        8192 |       81920 | data_node_1
+       16384 |       32768 |        8192 |       57344 | data_node_2
+           0 |        8192 |           0 |        8192 | 
+(3 rows)
+
```

Comment:
With explicit figures like this, isn't it going to change often even for changes that are not related to the function? What are we really testing here? That we can call the function? That we return these specific sizes? Something else?